src/porepy/models/contact_mechanics_biot_model.py

```diff
@@ -0,0 +1,462 @@
+"""
+This is a setup class for solving the biot equations with contact between the fractures.
+
+The domain $[0, 2]\times[0, 1]$ with six fractures. We do not consider any fluid, and
+solve only for the linear elasticity coupled to the contact
+
+NOTE: This module should be considered an experimental feature, which will likely
+undergo major changes (or be deleted).
+
+"""
+import numpy as np
+import porepy as pp
+
+import porepy.models.contact_mechanics_model as contact_model
+from porepy.utils.derived_discretizations import implicit_euler as IE_discretizations
+
+
+class ContactMechanicsBiot(contact_model.ContactMechanics):
+    def __init__(self, mesh_args, folder_name):
+        super().__init__(mesh_args, folder_name)
+
+        # Temperature
+        self.scalar_variable = "p"
+        self.mortar_scalar_variable = "mortar_" + self.scalar_variable
+        self.scalar_coupling_term = "robin_" + self.scalar_variable
+        self.scalar_parameter_key = "flow"
+
+        # Scaling coefficients
+        self.scalar_scale = 1
+        self.length_scale = 1
+
+        # Time
+        # The time attribute may be used e.g. to update BCs.
+        self.time = 0
+        self.time_step = 1 * self.length_scale ** 2
+        self.end_time = self.time_step * 1
+
+        # Initial scalar value
+        self.initial_scalar = 0
+
+        # Whether or not to subtract the fracture pressure contribution for the contact
+        # traction. This should be done if the scalar variable is pressure, but not for
+        # temperature. See assign_discretizations
+        self.subtract_fracture_pressure = True
```

Comment:
I have considered it. I think it might increase the risk of messing things up, but it would be neater. What do you think, @keileg?

Reply:
It's neater to identify it with the Biot alpha, unless someone gets confused. The approach we've taken lately is not to risk confusion. However, making it into a multiplication factor, possibly 0, rather than a boolean makes a lot of sense. I will try to make the change before merging, feel free to bug me if I drop it.